openpype/hosts/fusion/api/plugin.py

```diff
@@ -0,0 +1,262 @@
+from copy import deepcopy
+import os
+
+from openpype.hosts.fusion.api import (
+    get_current_comp,
+    comp_lock_and_undo_chunk,
+)
+
+from openpype.lib import (
+    BoolDef,
+    EnumDef,
+)
+from openpype.pipeline import (
+    legacy_io,
+    Creator as NewCreator,
```

Comment:
This does not have to be named `NewCreator`.
```suggestion
    Creator,
```
Need changes in whole file.